src/handlers/modal-submit.js

```diff
@@ -0,0 +1,51 @@
+const { formatPRMessage } = require('../utils/pr-formatter');
+
+async function handleModalSubmit(interaction) {
+  if (interaction.customId.startsWith('pr-template-modal-')) {
+    const channelId = interaction.customId.replace('pr-template-modal-', '');
+    const prUrl = interaction.fields.getTextInputValue('urlInput');
+    const title = interaction.fields.getTextInputValue('titleInput');
+    const overview = interaction.fields.getTextInputValue('overviewInput');
+    const testing = interaction.fields.getTextInputValue('testingInput');
```

Comment:
Maybe change this to "howToTest" and "howToTestInput", testing looks weird hehe

Reply:
check